src/app/admin/organisation/organisation-detail/organisation-detail.component.ts

```diff
@@ -100,6 +104,10 @@ export class OrganisationDetailComponent implements OnInit, OnChanges, OnDestroy
     this.deleteDialogSubscription = this.deleteDialogService.showSimpleDialog().subscribe(response => {
       if (response) {
         this.organisationService.delete(this.organisation.id).subscribe(response => {
+          if (response instanceof HttpErrorResponse) {
```

Comment:
Jeg ville have denne logik som subscribe's error callback, men handleError er disabled for put/post i restService.